docs/general-reference/managing-your-apps.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Managing Your Apps
+layout: en
+permalink: docs/general-reference/managing-your-apps/
+---
+
+<ul>
+	<li><a href="#prepare-your-application">Prepare Your Application</a></li>
+	<li><a href="#create-new-project">Create a New Project</a></li>
+	<li><a href="#app-versions">Manage App Versions</a></li>
+</ul>
+
+<!-- from faq -->
+<h3 id="prepare-your-application">Prepare Your Application</h3>
+
+To test your application on TestObject you will need to follow some guidelines when building your application. For iOS applications, you have to export your IPA file as an Ad Hoc Deployment. Check out this <a href="/docs/guides/creating-ipa/">short guide</a> on how to do it! For Android applications, you will simply need to build a debuggable version of your application.
+
+<!-- info for whitelisting ip's -->
+
+<h3 id="create-new-project">Create a New Project</h3>
+
+If this isn't the first project on your account, you will first need to select the "New App" button from your home screen:
```

Comment:
I think most people looking at this guide might be in the other situation: nothing uploaded and they want some info on how to proceed (which option to select). Let's give more weight to this scenario, while still mentioning the second to make sure people can follow.